util/ppc-asm.h

```diff
@@ -0,0 +1,390 @@
+/* PowerPC asm definitions for GNU C.
+
+Copyright (C) 2002-2020 Free Software Foundation, Inc.
+
+This file is part of GCC.
+
+GCC is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License as published by the Free
+Software Foundation; either version 3, or (at your option) any later
+version.
```

Comment:
This appears to be permissible copying. (Internal ref: https://fburl.com/wiki/s1ewaih5)

However, I think it belongs under third-part/gcc/ or even port/ instead of util/.